lib/stomp_config.dart

```diff
@@ -81,7 +85,8 @@ class StompConfig {
           int heartbeatIncoming,
           int heartbeatOutgoing,
           Duration connectionTimeout,
-          Map<String, dynamic> connectHeaders,
+          Map<String, dynamic> stompConnectHeaders,
```

Comment:
Because of the type difference between `stompConnectHeaders`/`webSocketConnectHeaders` parameters in copyWith and their definition in StompConfig, I get the following error  :
```
 type '_InternalLinkedHashMap<String, dynamic>' is not a subtype of type 'Map<String, String>'
```

You can try it by adding the following test :
```dart
test('copyWith', () {
      config = config.copyWith(
        webSocketConnectHeaders: {
          "X-Header": "some string"
        },
        stompConnectHeaders: {
          "X-Header": "some string"
        });
    });
```